### PR TITLE
Use commercial fonts in website everywhere

### PR DESCRIPTION
**Pull request by ishantsolanki**

### CODE REVIEW

File: assets/css/_custom-resets.scss [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 @mixin title-font {
-  font-family: 'Molot', sans-serif;
+  font-family: 'व्यावहारिकNL', 'Molot', sans-serif;
   font-weight: normal;
-  -webkit-font-feature-settings: 'dlig' 1, 'liga' 1, 'lnum' 1, 'kern' 1;
-  -moz-font-feature-settings: 'dlig' 1, 'liga' 1, 'lnum' 1, 'kern' 1;
-  -o-font-feature-settings: 'dlig' 1, 'liga' 1, 'lnum' 1, 'kern' 1;
+  -webkit-font-feature-settings: 'liga' 1, 'lnum' 1, 'kern' 1;
+  -moz-font-feature-settings: 'liga' 1, 'lnum' 1, 'kern' 1;
+  -o-font-feature-settings: 'liga' 1, 'lnum' 1, 'kern' 1;
```
Comment (ishantsolanki) [inline assets/css/_custom-resets.scss]: The `dlig` incorrect font rendering was due to these lines. Removing the 'dlig' option fixes it. I am not sure what it does.

Comment (missingfaktor) [reply assets/css/_custom-resets.scss]: Thanks. Here is the fix from Fabrizio, FWIW: https://twitter.com/fabrizioschiavi/status/996026899045511168.

**Comment by missingfaktor** [comment]
The code appears in Ubuntu Mono. We should check why our commercial font is not being picked up. 

screen shot 2018-05-14 at 23 49 09

**Comment by missingfaktor** [comment]
Super cool! Cheers!

**Pull request merged**